### ISSUE TITLE
MixedOps-NI-MultiReconnect-6k-39m : NPE in Fee Calculation

### ISSUE THREAD

**Issue by anighanta** [hashgraph/hedera-services#1730]
### Description

All nodes fails with NPE in `hgcaa.log` when calculating fee. `Map<SubType, FeeData>` for `SubType.DEFAULT` is null as the feeSchedules are not loaded.

```2021-07-02 06:08:40.772 ERROR 100  ServicesMain - Fatal precondition violated in HederaNode#0!
java.lang.IllegalStateException: Could not initialize system files!
	at com.hedera.services.ServicesMain.initSystemFiles(ServicesMain.java:194) ~[HederaNode.jar:?]
	at com.hedera.services.ServicesMain.contextDrivenInit(ServicesMain.java:152) ~[HederaNode.jar:?]
	at com.hedera.services.ServicesMain.init(ServicesMain.java:97) [HederaNode.jar:?]
	at com.swirlds.platform.SwirldsPlatform.initializeFirstStep(SwirldsPlatform.java:878) [swirlds-platform-core-0.16.0.jar:?]
	at com.swirlds.platform.Browser.startPlatforms(Browser.java:982) [swirlds-platform-core-0.16.0.jar:?]
	at com.swirlds.platform.Browser.main(Browser.java:364) [swirlds-platform-core-0.16.0.jar:?]
Caused by: java.lang.NullPointerException
	at com.hedera.services.fees.calculation.AutoRenewCalcs.constantFeeFrom(AutoRenewCalcs.java:131) ~[HederaNode.jar:?]
	at com.hedera.services.fees.calculation.AutoRenewCalcs.setCryptoAutoRenewPriceSeq(AutoRenewCalcs.java:65) ~[HederaNode.jar:?]
	at com.hedera.services.fees.calculation.UsageBasedFeeCalculator.init(UsageBasedFeeCalculator.java:101) ~[HederaNode.jar:?]
	at com.hedera.services.files.sysfiles.CurrencyCallbacks.lambda$feeSchedulesCb$1(CurrencyCallbacks.java:54) ~[HederaNode.jar:?]
	at com.hedera.services.state.initialization.HfsSystemFilesManager.loadProtoWithSupplierFallback(HfsSystemFilesManager.java:247) ~[HederaNode.jar:?]
	at com.hedera.services.state.initialization.HfsSystemFilesManager.loadFeeSchedules(HfsSystemFilesManager.java:145) ~[HederaNode.jar:?]
	at com.hedera.services.state.initialization.SystemFilesManager.loadObservableSystemFiles(SystemFilesManager.java:50) ~[HederaNode.jar:?]
	at com.hedera.services.state.logic.NetworkCtxManager.loadObservableSysFilesIfNeeded(NetworkCtxManager.java:94) ~[HederaNode.jar:?]
	at com.hedera.services.ServicesMain.initSystemFiles(ServicesMain.java:192) ~[HederaNode.jar:?]
	... 5 more
```

### Steps to reproduce

Run test `MixedOps-NI-MultiReconnect-6k-39m`

### Hedera network

other

### Version

master, v0.16.0

### Operating system

Linux

**Comment by ljianghedera** [comment]
Any tests that need to load saved state file failed for the same reason.

**Comment by tinker-michaelj** [comment]
Believed closed by PR #1740

**Comment by ljianghedera** [comment]
Confirmed close by #1740
